requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib>=3.7.3
```

Comment:
There should be strict equality here, in case some package gets updated and something breaks

Reply:
![reprojection_error_for_each_pair](https://github.com/AnNyiiik/AlignmentOfUAVImages/assets/114094098/326ee9f6-9459-4aa1-adba-83e31e385c05)

![reprojection_error_for_each_pair](https://github.com/AnNyiiik/AlignmentOfUAVImages/assets/114094098/d0f54d73-d3bd-42f9-8d8b-026bca82b5bf)

Reply:
![Figure_1](https://github.com/AnNyiiik/AlignmentOfUAVImages/assets/114094098/8026519e-4023-4cea-8225-9dfc9e8a5114)